### ISSUE TITLE
Missing new relics from 25.3

### ISSUE THREAD

**Issue by AlyoshaVasilieva** [WFCD/warframe-items#90]
For example https://warframe.fandom.com/wiki/Axi_M1

According to [wiki](https://warframe.fandom.com/wiki/Category:Update_25) the new relics are

* Axi D1
* Axi G2
* Axi L5
* Axi M1
* Axi P1
* Lith L1
* Lith W1
* Meso A2
* Meso K2
* Meso N7
* Neo R2
* Neo S12
* Neo Z3

**Comment by TobiTenno** [comment]
DE hasn't added them in their manifests. Sorry.

**Comment by AlyoshaVasilieva** [comment]
I grabbed ExportRelicArcane_en.json from the mobile endpoint and it has the new relics, though I don't actually know how this project works and if that has enough info. It seems the endpoint I grabbed from is different (the one listed in `http://origin.warframe.com/origin/00000000/PublicExport/index_en.txt.lzma`)

ex:

```json
    {
      "uniqueName": "/Lotus/Types/Game/Projections/T4VoidProjectionWukongPrimeCBronze",
      "name": "AXI M1 RELIC",
      "codexSecret": false,
      "description": "An artifact containing Orokin secrets. It can only be opened through the power of the Void.",
      "relicRewards": [
        {
          "rewardName": "/Lotus/StoreItems/Types/Recipes/WarframeRecipes/MiragePrimeBlueprint",
          "rarity": "RARE",
          "tier": 0,
          "itemCount": 1
        },
      ]
    },
```

[rewards trimmed]

Anyway, hopefully DE updates manifests soon.

**Comment by TobiTenno** [comment]
we don't touch origin.warframe.com, we use [these](https://github.com/WFCD/warframe-items/blob/development/config/endpoints.json)

**Comment by Svarr** [comment]
It's not only the relics that are missing, it's the whole new prime content. And since the official app already contains those items, I can only assume that they switched APIs. Maybe _MobileExport_ will be shut down soon?

Btw, what is _origin.warframe.com_?

**Comment by TobiTenno** [comment]
if they just got them today, then it's just not processed yet. I checked yesterday when I commented, and the new primes were not on the mobile app yet.

origin.warframe.com is one of their private data sources that we don't touch cause we've not been authorized to grab from there.

**Comment by Svarr** [comment]
Wukong Prime is listed in the codex but not in the manifest json.

**Comment by TobiTenno** [comment]
we can't really change that....

**Comment by Kaptard** [comment]
Closed in favor of #91
